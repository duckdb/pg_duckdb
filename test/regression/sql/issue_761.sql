SET duckdb.force_execution = TRUE;

DO $$
DECLARE
    objtype text;
BEGIN
    FOR objtype IN VALUES ('toast table'), ('index column'), ('sequence column'),
        ('toast table column'), ('view column'), ('materialized view column')
    LOOP
        BEGIN
            PERFORM pg_get_object_address(objtype, '{one}', '{}');
        EXCEPTION WHEN invalid_parameter_value THEN
            RAISE WARNING 'error for %: %', objtype, sqlerrm;
        END;
    END LOOP;
END;
$$;
