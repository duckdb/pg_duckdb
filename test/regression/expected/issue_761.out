SET duckdb.force_execution = TRUE;
DO $$
DECLARE
    objtype text;
BEGIN
    FOR objtype IN VALUES ('toast table'), ('index column'), ('sequence column'),
        ('toast table column'), ('view column'), ('materialized view column')
    LOOP
        BEGIN
            PERFORM pg_get_object_address(objtype, '{one}', '{}');
        EXCEPTION WHEN invalid_parameter_value THEN
            RAISE WARNING 'error for %: %', objtype, sqlerrm;
        END;
    END LOOP;
END;
$$;
WARNING:  error for toast table: unsupported object type "toast table"
WARNING:  error for index column: unsupported object type "index column"
WARNING:  error for sequence column: unsupported object type "sequence column"
WARNING:  error for toast table column: unsupported object type "toast table column"
WARNING:  error for view column: unsupported object type "view column"
WARNING:  error for materialized view column: unsupported object type "materialized view column"
ERROR:  (PGDuckDB/Duckdb_EndCustomScan_Cpp) Executor Error: SAVEPOINT is not supported in DuckDB
CONTEXT:  PL/pgSQL function inline_code_block line 5 at FOR over SELECT rows
